src/app/games/conspirateurs/ConspirateursMinimax.ts

```diff
@@ -0,0 +1,111 @@
+import { Coord } from 'src/app/jscaip/Coord';
+import { Minimax } from 'src/app/jscaip/Minimax';
+import { NodeUnheritance } from 'src/app/jscaip/NodeUnheritance';
+import { Player } from 'src/app/jscaip/Player';
+import { MGPFallible } from 'src/app/utils/MGPFallible';
+import { ConspirateursMove, ConspirateursMoveDrop, ConspirateursMoveJump, ConspirateursMoveSimple } from './ConspirateursMove';
+import { ConspirateursNode, ConspirateursRules } from './ConspirateursRules';
+import { ConspirateursState } from './ConspirateursState';
+
+export class ConspirateursMinimax extends Minimax<ConspirateursMove, ConspirateursState> {
+    public getListMoves(node: ConspirateursNode): ConspirateursMove[] {
+        if (node.gameState.turn < 40) {
+            return this.getListMovesBeforeDrop(node.gameState);
+        } else {
+            return this.getListMovesAfterDrop(node.gameState);
+        }
+    }
+    private getListMovesBeforeDrop(state: ConspirateursState): ConspirateursMoveDrop[] {
+        const moves: ConspirateursMoveDrop[] = [];
+        for (let y: number = ConspirateursState.CENTRAL_ZONE_TOP_LEFT.y;
+            y <= ConspirateursState.CENTRAL_ZONE_BOTTOM_RIGHT.y;
+            y++) {
```

Comment:
for (a, b, c) should have a, b and c starting either on the same row or on the same column (column here) ?
And multiline loop should have \n before { ? :p ?
(c'est ça que le linter devrais dire héhé :D )
A appliquer sur le suivant aussi please :p

Reply:
arr y'a pas d'option pour dire au linter de forcer cette indentation, et il autorise pas l'indentation voulue D: Je désactive la règle de lintage localement, mais ça pue un peu. Une autre idée sinon ?

Reply:
constantiser maximumX et maximumY et monoliniser ça joliment ^^ ?

Reply:
yep pas con, fait